makefile

```diff
@@ -303,7 +303,7 @@ download:
 	$(PIP) install $(PROJECT)
 
 jupyter: $(JUPYTER)
-	$(BIN)/jupyter-notebook $(args) --notebook-dir=$(NOTEBOOK_CHECKS)
+	$(BIN)/jupyter-notebook $(args) --notebook-dir=./notebooks
```

Comment:
why?

Reply:
because I want access to checks & examples, not only checks